index.js

```diff
@@ -86,6 +86,9 @@ function Peer (opts) {
   self._interval = null
 
   self._pc = new (self._wrtc.RTCPeerConnection)(self.config, self.constraints)
+  if (self._isChromium || opts.wrtc) {
```

Comment:
I thought Chromium supports Promise-based API for quite a while now. What is the issue there?
Also I think node-webrtc should also support Promise-based API. So maybe feature detection as in `getStats()` instead?

Reply:
Chromium shows the same behaviour as Safari with constraints (it ignores offerToReceiveAudio etc). Apparently this is intended behaviour as transceivers are meant to replace these deprecated options. This only happens when using the Promise API.

However, due to #95 and Chrome's lack of transceiver support, we have no other way of setting these constraints except through the callback API.

`electron-webrtc` doesn't support the Promise API. I suppose node-webrtc might be fine to use Promises.

Reply:
I get some false positives on other browsers with those: So I've used `self._wrtc.electronDaemon`, which should be foolproof.

Reply:
Doesn't look pretty, but if it works, we can live with it for a while.

Reply:
Hopefully we can move to transceivers eventually and remove all callback usage. 👍